src/main/java/org/spongepowered/mod/entity/SpongeHorseColor.java

```diff
@@ -32,4 +32,9 @@ public SpongeHorseColor(int color, String name) {
         super(color, name);
     }
 
+    @Override
+    public boolean isFlowerPot() {
+        return false == true;
```

Comment:
This is the correct and most efficient logic check. Please be consistent and apply `false == true` to all other methods.